google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java

```diff
@@ -174,14 +169,14 @@ public Table updateTable(String datasetName, String tableName, String newDescrip
   // [TARGET delete(String, DatasetDeleteOption...)]
   // [VARIABLE "my_dataset_name"]
   public Boolean deleteDataset(String datasetName) {
-    // [START bigquery_java_untracked]
+    // [START ]
```

Comment:
👌 Looks a little strange not to have the region tag, but makes sense to keep it this way since it's not included in cloud.google.com.